src/main/java/dan200/computercraft/shared/peripheral/generic/data/ItemData.java

```diff
@@ -83,4 +149,19 @@ private static ITextComponent parseTextComponent( @Nonnull INBT x )
             return null;
         }
     }
+
+    @Nonnull
+    private static List<Map<String, Object>> enchantMapToList( Map<Enchantment, Integer> rawEnchants )
+    {
+        List<Map<String, Object>> enchants = new ArrayList<>( rawEnchants.size() );
+        rawEnchants.forEach( ( enchantment, level ) ->
```

Comment:
Probably easier just to do a `for` loop.

Reply:
I did that because IntelliJ say me:
`foreach not applicable to type 'java.util.Map<net.minecraft.enchantment.Enchantment,java.lang.Integer>'`

Reply:
Oh, use `.entrySet()`.